pkg/apis/postgres-operator.crunchydata.com/v1beta1/pgbouncer_types.go

```diff
@@ -26,7 +26,12 @@ type PGBouncerConfiguration struct {
 
 	// Settings that apply to the entire PgBouncer process.
 	// More info: https://www.pgbouncer.org/config.html
+	//
+	// # Logging
+	// +kubebuilder:validation:XValidation:rule=`!has(self.logfile) || self.logfile.endsWith('.log')`,message=`logfile config must end with '.log'`
+	//
 	// +optional
+	// +mapType=granular
```

Comment:
👍🏻 I want these granular/set/atomic markers on all the new collections we add to the API.